x/staking/types/params.go

```diff
@@ -24,35 +24,40 @@ const (
 
 	// Default maximum entries in a UBD/RED pair
 	DefaultMaxEntries uint16 = 7
+
+	DefaultHistoricalEntries uint16 = 0
```

Comment:
Add a comment regarding the default choice. Eg: `"Default value is 0 as it's only required for apps that implement the IBC module"`